lib/configSchema.js

```diff
@@ -96,22 +109,42 @@ const schema = {
       additionalProperties: false,
     },
 
-    package: {
+    package: packageSchema,
+
+    layers: {
       type: 'object',
-      properties: {
-        individually: { type: 'boolean' },
-        path: { type: 'string' },
-        artifact: { type: 'string' },
-        exclude: { type: 'array', items: { type: 'string' } },
-        include: { type: 'array', items: { type: 'string' } },
-        excludeDevDependencies: { type: 'boolean' },
+      additionalProperties: {
+        type: 'object',
+        properties: {
+          allowedAccounts: {
+            type: 'array',
+            items: {
+              oneOf: [
+                { type: 'integer', minimum: 100000000000, maximum: 999999999999 },
+                { type: 'string', pattern: '^\\d{12}$' },
```

Comment:
I think pattern should be as here: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-layerversionpermission.html#cfn-lambda-layerversionpermission-principal